### PR TITLE
Riena

### PR DESCRIPTION
**Pull request by riena42**
entities without user entities

### CODE REVIEW

**Review by ponyopm — APPROVED** [review]
Good job overall refactoring the entities.  I like the simplicity that comes from not having an entity factory for simple entities.

**Review by anthonymountain — DISMISSED** [review]
Adheres to principles of clean architecture and follows conventions established with other use case classes, looks good. Refactoring of entities seems to be in line with agreed upon changes. Deletion of factories was done well, seems to be in order. Good work.

File: src/main/java/entity/Album.java
Comment (Descartes-AR) [inline src/main/java/entity/Album.java:1]: This implementation of Album is perfectly in line with the Spotify Web API documentation and perfectly captures all attributes required by our respective usecases. The edits are well constructed to replace the previous entity and are easy to understand. The integration of instance variables using genres and Track objects will help us transfer the json object data easily in a format that we can modify and interact with ease. Good Job!

**Review by Descartes-AR — APPROVED** [review]
The implementation of changes to the entities based on our discussion and read through of the Spotify Web API documentation seems to be flawless and will be critical to our progress on the app. The refinement of the code is apparent and follows clean architecture perfectly. Great work.

**Pull request merged**